shadow.c

```diff
@@ -33,9 +33,9 @@ void initialize_pw_backend(int argc, char **argv) {
 		swaylock_log_errno(LOG_ERROR, "Unable to drop root");
 		exit(EXIT_FAILURE);
 	}
-	if (setuid(0) != -1) {
+	if (setuid(0) != -1 || setgid(0) != -1) {
```

Comment:
nit: The we handle gid and uid separately above. No opinion as to whether it should be one or two if's, but consistency is nice. :)

Reply:
So you mean creating a second if statement for setgid?

Reply:
I think it's fine. Thanks!